pandas/io/parsers.py

```diff
@@ -999,6 +999,8 @@ def _failover_to_python(self):
 
     def read(self, nrows=None):
         if nrows is not None:
+            nrows = _validate_integer('nrows', nrows)
```

Comment:
I originally didn't have this line there, but then I was getting the following error:
```TypeError: '>=' not supported between instances of 'int' and 'str'``` vs
```ValueError: 'nrows' must be an integer >=0```